File: Semester 4/Programming and Projection Enironments/mpp-proiect-java/Proiect_MPP/AirportPersistence/src/main/java/root/persistence/airports/IAirportRepository.java
package root.persistence.airports;

import root.model.Airport;
import root.persistence.Repository;

import java.util.Collection;

public interface IAirportRepository extends Repository<Airport, Integer> {
    Collection<Airport> getAirportAfterName(String name);
    Collection<Airport> getAirportsInCity(String cityName);
}

File: Semester 4/Programming and Projection Enironments/mpp-proiect-csharp/Proiect_MPP/persistence/airports/IAirportRepository.cs
﻿using model;
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace persistence.airports
{
    public interface IAirportRepository : Repository<Airport, int>
    {
        List<Airport> getAirportAfterName(string name);
        List<Airport> getAirportsInCity(string cityName);
    }
}
